spec/Memory_Consistency.tex

```diff
@@ -120,6 +847,43 @@
 statements block.
 \end{chapelexample}
 
+\begin{chapelexample}{atomicSpinWait.chpl}
+Atomic variables provide an alternative means to spin-wait.
+For example:
+
+\begin{chapel}
+var x: atomic int;
+cobegin {
+  while x.read() != 1 do ;  // spin wait - monopolizes processor
```

Comment:
the Chapel tasking model would allow the spinner to monopolize the CPU given this code, preventing progress and hanging the program; better would be to use a waitFor